docs/releases.rst

```diff
@@ -292,10 +306,19 @@ Upgrading
 `````````
 
 Upgrade should run fine as long as you generate a new compose & mailu.env and then reapply custom config settings to mailu.env.
+Carefully read the :ref:`configuration page <common_cfg>` to check what old settings have been removed. If a setting is not listed anymore
+on the :ref:`configuration page <common_cfg>`, then this setting has been removed.
+
+If you use Fail2Ban, then the Fail2Ban intructions have been improved. It is **mandatory** to remove your Fail2Ban config
+and re-apply it using the instructions from :ref:`updated Fail2Ban documentation <Fail2Ban>`.
 
-If you use Fail2Ban, then the Fail2Ban intructions have been improved. It is mandatory to remove your Fail2Ban config and re-apply it using the instructions from :ref:`updated Fail2Ban documentation <Fail2Ban>`.
+If you use overrides for Rspamd, then please note that overrides are now placed in the location `/overrides` in the rspamd container.
+If you use your own map files, change the location to /override/myMapFile.map in the corresponding rspamd conf file.
 
 To use the new autoconfig endpoint and Mailu RESTFul API, you may need to update your reverse proxy config.
+If you use ``TLS_FLAVOR=letsencrypt``, add autoconfig.myhostname.com to the setting ``HOSTNAMES=`` in mailu.env to generate a certifficate for the autoconfig endpoint as well.
```

Comment:
Maybe hint at mta-sts in the FAQ too?

Reply:
Yes. That is a good idea. I will submit yet another commit.

Reply:
oh and s/certifficate/certificate/

Reply:
Fixed the typo and I added a hint to have a look at mta-sts. I hope my of mta-sts in layman terms  is correct.